src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
```

Comment:
>FullId [](start = 53, length = 6)

shall we have a more strict check to make sure tagRule, rule and category don't conflict?